### PR TITLE
docs: added `Add assets into Astar Portal` section under XCM tab

### PR DESCRIPTION
**Pull request by impelcrypto**
* docs: added docs for adding XCM assets to our portal
* staging URL: https://astar-docs--pr91-docs-added-xcmp-inte-zap7zftl.web.app/docs/xcm/integration/add-to-portal/

### CODE REVIEW

File: docs/xcm/integration/add-to-portal.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+# Add assets into Astar Portal
+
+Once a channel is opened between parachains and assets have been registered, developers can create a PR to [our portal](https://github.com/AstarNetwork/astar-apps) for adding assets to our portal. Here’s what you need to know about the integration of the XCM assets into our portal.
+
+## Define the parachains information
+
+1. Add the `Chain name` at the [Chain](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
+2. Add the `parachain id` at the [parachainIds](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
```
Comment (andabakus) [inline docs/xcm/integration/add-to-portal.md:12]: minor typo - should link to https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L17

**Comment by andabakus** [comment]
General comment: should we ask or specify for some extra documentation that we expect to be submitted with PR - (like proof that test have passed - like screenshots, logs from console and Tx Ids of test transactions) ?  

Of course, first couple of PRs will definitely be closely moderated by Astar Team, but just as an idea if it is possible already?

**Comment by impelcrypto** [comment]
@andabakus Great idea! Perhaps we can ask for submitting the Subscan link for the transaction details to be submitted with PR.

File: docs/xcm/integration/add-to-portal.md (hunk 1 above)
Comment (impelcrypto) [reply docs/xcm/integration/add-to-portal.md]: I've fixed in [this](https://github.com/AstarNetwork/astar-docs/pull/91/commits/b7859f20a3b5454774885619a722fe287184b438) commit. Thank you!

**Comment by impelcrypto** [comment]
@andabakus I've added [Requirement for creating a PR](https://astar-docs--pr91-docs-added-xcmp-inte-zap7zftl.web.app/docs/xcm/integration/add-to-portal/#requirement-for-creating-a-pr) section.

File: docs/xcm/integration/add-to-portal.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+sidebar_position: 5
+---
+
+# Add assets into Astar Portal
+
+Once a channel is opened between parachains and assets have been registered, developers can create a PR to [our portal](https://github.com/AstarNetwork/astar-apps) for adding assets to our portal. Here’s what you need to know about the integration of the XCM assets into our portal.
+
+## Define the parachains information
+
+1. Add the `Chain name` at the [Chain](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
```
Comment (bobo-k2) [inline docs/xcm/integration/add-to-portal.md:11]: This is not a documentation comment it is an idea for a small code refactoring. Lets move point 1, 2 and 3 to parachain repository implementation. In that case we will have a single class where changes needs to be implemented. What do you think?

**Review by bobo-k2 — APPROVED** [review]
LGTM

File: docs/xcm/integration/add-to-portal.md (hunk 2 above)
Comment (impelcrypto) [reply docs/xcm/integration/add-to-portal.md:11]: Cool! Could you create a PR because I'm not available today.

Comment (bobo-k2) [reply docs/xcm/integration/add-to-portal.md:11]: Sure. I'll do it later today or tomorrow

**Pull request merged**